.gitignore

```diff
@@ -0,0 +1,2 @@
+Cargo.lock
```

Comment:
Including the lock file ensures any dependency upgrade (including minor bumps) will require a change in this repository too. Are we sure we want to do this?